dev/source/docs/ardupilot-on-linux-init-scripts.rst

```diff
@@ -0,0 +1,38 @@
+.. _ardupilot-on-linux-init-scripts:
+
+============================
+Linux Initialization Scripts
+============================
+
+
+Example Systemd Unit File
+=========================
+
+Adjust the following and place in /etc/systemd/system/arduplane.service
+
+::
+
+    [Unit]
+    Description=ArduPlane Service
+    Conflicts=arducopter.service ardupilot.service ardurover.service
+
+    [Service]
+    EnvironmentFile=/etc/default/arduplane
+
+    # Lock CPU governor to top speed for reduced latency
+    ExecStartPre=/bin/bash -c "/bin/echo performance | tee /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor"
+
+    # Steer any IRQs Ex: IRQ#30 to CPU 3
+    ExecStartPre=/bin/bash -c "echo 3 > /proc/irq/30/smp_affinity_list"
```

Comment:
need some explanation for this one as this isn't strait forward for all, or a link to some explanation

Reply:
Got it. The need for this might be mostly specific to my board, since I'm running multiple real-time processes. We could remove it if preferred.

Reply:
This is no longer rendering correctly, one moment please.